package.json

```diff
@@ -16,7 +16,7 @@
     "test": "tap test/*.js"
   },
   "bugs": "http://github.com/creationix/jsonparse/issues",
-  "engines": ["node >= 0.2.0"],
+  "engines": ["node >= 4.8.2"],
```

Comment:
It seems that `String.fromCharCode(high,low);` works. so i've returned 0.2.0 version of node.